RequestService/RequestService.Repo/Repository.cs

```diff
@@ -2098,62 +2098,59 @@ public async Task<int> OpenJobCount(IEnumerable<int> groups)
                 );
         }
 
-        public async Task<Chart> GetActivitiesByMonth(int groupId)
+        public async Task<List<DataItem>> GetActivitiesByMonth(int groupId, DateTime minDate)
```

Comment:
I like that you're padding in a minDate.
Remove `DateTime dt = DateTime.UtcNow.Date.AddYears(-1);` ?